eacl/target.go

```diff
@@ -6,17 +6,44 @@ import (
 
 	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
 	v2acl "github.com/nspcc-dev/neofs-api-go/v2/acl"
+	neofscrypto "github.com/nspcc-dev/neofs-sdk-go/crypto"
+	neofsecdsa "github.com/nspcc-dev/neofs-sdk-go/crypto/ecdsa"
 )
 
 // Target is a group of request senders to match ContainerEACL. Defined by role enum
 // and set of public keys.
 //
 // Target is compatible with v2 acl.EACLRecord.Target message.
+//
+// Filter should be created using one of the constructors.
 type Target struct {
 	role Role
 	keys [][]byte
 }
 
+// TargetByRole returns Target for specified role. Use TargetByRole in [Record]
+// to direct it to subjects with the given role in NeoFS.
+func TargetByRole(role Role) Target { return Target{role: role} }
+
+// TargetByPublicKeys returns Target for specified binary-encoded public keys.
+// Use TargetByPublicKeys in [Record] to direct it to subjects authenticated by
+// one of the given keys in NeoFS. See also [EncodeECDSAPublicKey]. The ks must
+// not be empty.
+func TargetByPublicKeys(ks [][]byte) Target { return Target{keys: ks} }
```

Comment:
it can, but if user has already encoded keys then redundant decode-encode will be done

if u ask why dont we also provide `TargetByECDSAPublicKeys([]ecdsa.PublicKey)` - i dont mind to add it

Reply:
#613.